pc/src/main.rs

```diff
@@ -110,6 +93,32 @@ impl GameState {
 
 async fn run_game() {
 	let mut game_state = GameState::new();
+	let mut events = game_state.input_players.get_events();
+	let mut no_player = true;
+
+	match &mut game_state.game_run {
+		None => todo!("no game run"),
+		Some(ref mut game_run) => {
+			while no_player {
+				events = game_state.input_players.get_events();
+				for event in events {
+					if event.is_some() {
+						no_player = false;
+						break;
+					}
+				}
+				next_frame().await;
+			}
+
+			let players = Players::init();
+			for player in players.possible_players {
+				player.send_events(ToPypadeGameEvent::NewLevel(
+					game_run.level.cards.clone()
+				));
+				println!("send cards to player");
```

Comment:
The events are already printed if the log level is set to debug (`RUST_LOG=debug cargo run`).
Since we using the log crate you should always use `info!`, `warn!`, `error!`, `debug!` and `trace!` instead of `print` directly.

```suggestion
```